client/resource_group/controller/controller.go

```diff
@@ -192,6 +197,7 @@ func (c *ResourceGroupsController) Start(ctx context.Context) {
 		for {
 			select {
 			case <-c.loopCtx.Done():
+				c.resetMetrics()
```

Comment:
Do we need to reset `resourceGroupStatusGauge` for each resource group when it's deleted from the client?

Reply:
Yes, I do it in `cleanUpResourceGroup`